CMakeLists.txt

```diff
@@ -502,13 +504,8 @@ if(VELOX_ENABLE_COMPRESSION_LZ4)
   find_package(lz4 REQUIRED)
 endif()
 
-if(${VELOX_BUILD_MINIMAL_WITH_DWIO} OR ${VELOX_ENABLE_HIVE_CONNECTOR})
-  # DWIO needs all sorts of stream compression libraries.
-  #
-  # TODO: make these optional and pluggable.
-  find_package(ZLIB REQUIRED)
+if(VELOX_ENABLE_COMPRESSION_ZSTD)
```

Comment:
If VELOX_BUILD_MINIMAL_WITH_DWIO is ON and VELOX_ENABLE_COMPRESSION_ZSTD is OFF, we don't find the zstd package, is it the expected behavior?
I think if(${VELOX_BUILD_MINIMAL_WITH_DWIO} OR ${VELOX_ENABLE_HIVE_CONNECTOR}), we should set VELOX_ENABLE_COMPRESSION_ZSTD to true

Reply:
This logic exists here https://github.com/facebookincubator/velox/pull/13817/files#diff-1e7de1ae2d059d21e1dd75d5812d5a34b0222cef273b7c3a2af62eb747f9d20aR208